### PR TITLE
[REF] CONTRIBUTING: Add use of default label string in fields.

### PR DESCRIPTION
**Pull request by moylop260**
By default new api use name of field in label string.
Use this feature if is posible.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -400,6 +400,10 @@ class...
   (example: sale_order_line_ids)
 * `Many2One` fields should have `_id` as suffix
   (example: partner_id, user_id, ...)
+* If the name of field is the same to name of label string don't setting `string` parameter for fields of api v8.
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: Rephrase this to:

`If the technical name of the field (the variable name) is the same to the string of the label, don't put `string` parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized`

Comment (moylop260) [inline CONTRIBUTING.md]: @pedrobaeza 
Thank for you feedback.
Done!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -400,6 +400,9 @@ class...
   (example: sale_order_line_ids)
 * `Many2One` fields should have `_id` as suffix
   (example: partner_id, user_id, ...)
+* If the technical name of the field (the variable name) is the same to the string of the label, don't put `string` parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized.
```
Comment (pedrobaeza) [inline CONTRIBUTING.md]: You have to enclose the _ like code, because it's making the parser to interpret it as italics begin

Comment (moylop260) [inline CONTRIBUTING.md]: I don't understand
Can you help to clarify?
FYI I saw this: http://awesomescreenshot.com/0d954qsy30

Comment (pedrobaeza) [inline CONTRIBUTING.md]: Forget it, it was, only something of the preview. That doesn't happen in the full file view.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by nhomar** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:
